scaleway/resource_lb_frontend.go

```diff
@@ -153,14 +153,14 @@ func resourceScalewayLbFrontend() *schema.Resource {
 }
 
 func resourceScalewayLbFrontendCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	lbAPI, _, err := lbAPIWithZone(d, meta)
+	lbAPI, zone, err := lbAPIWithZone(d, meta)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	zone, lbID, err := parseZonedID(d.Get("lb_id").(string))
-	if err != nil {
-		return diag.FromErr(err)
+	lbID := expandID(d.Get("lb_id"))
```

Comment:
What happens if the provider and the zone of id are different ?

Reply:
The lb should be at the same zone